syntax = "proto3";

import "google/protobuf/empty.proto";

service NotifierService {
  rpc SendNotification (NotifierRequest) returns (google.protobuf.Empty);
}

message NotifierRequest {
  string notification_type = 1;
  string researcher = 2;
  string measurement_id = 3;
  string experiment_id = 4;
  string cipher_data = 5;
}
